packages/openapi-generator/src/cli-parser.ts

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env node
```

Comment:
[q] I doubt that this will execute with node because of the type annotations. So why do we have this shebang?

Reply:
This was a copy paste from the [OData](https://github.com/SAP/cloud-sdk-js/blob/main/packages/generator/src/cli-parser.ts) parser where a code reuse was difficult - but I will try to add something in the generator-commons.

Reply:
removed it - in the `cli.ts` which is the bin in the package.json it makes sense. Here not I think.